? contrib/include/wx/gizmos/editlbox_new.h
? contrib/src/gizmos/editlbox_new.cpp
Index: contrib/include/wx/gizmos/multicell.h
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/contrib/include/wx/gizmos/multicell.h,v
retrieving revision 1.12
diff -p -t -b -B -u -2 -r1.12 multicell.h
--- contrib/include/wx/gizmos/multicell.h	2006/04/26 16:27:16	1.12
+++ contrib/include/wx/gizmos/multicell.h	2007/02/09 03:32:15
@@ -61,12 +61,12 @@ public:
     wxMultiCellItemHandle( int row, int column, wxSize size, wxResizable style = wxNOT_RESIZABLE, wxSize weight = wxSize(1,1), int align = wxALIGN_NOT);
     wxMultiCellItemHandle( int row, int column, wxResizable style, wxSize weight = wxSize(1,1), int align = wxALIGN_NOT);
-    int             GetColumn();
-    int             GetRow();
-    int             GetWidth();
-    int             GetHeight();
-    wxResizable     GetStyle();
-    wxSize          GetLocalSize();
-    int             GetAlignment();
-    wxSize          GetWeight();
+    int             GetColumn() const;
+    int             GetRow() const;
+    int             GetWidth() const;
+    int             GetHeight() const;
+    wxResizable     GetStyle() const;
+    wxSize          GetLocalSize() const;
+    int             GetAlignment() const;
+    wxSize          GetWeight() const;
 
 private:
@@ -102,5 +102,5 @@ public:
 private:
     void GetMinimums();
-    int Sum(int *array, int x);
+    int Sum(int *array, int x) const;
 
 private:
@@ -136,9 +136,9 @@ public:
 
     void Resize(int numRows, int numCols);
-    int MaxRows()
+    int MaxRows() const
     {
         return m_maxRows;
     };
-    int MaxCols()
+    int MaxCols() const
     {
         return m_maxCols;
Index: contrib/src/gizmos/multicell.cpp
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/contrib/src/gizmos/multicell.cpp,v
retrieving revision 1.17
diff -p -t -b -B -u -2 -r1.17 multicell.cpp
--- contrib/src/gizmos/multicell.cpp	2006/10/16 09:29:31	1.17
+++ contrib/src/gizmos/multicell.cpp	2007/02/09 03:32:11
@@ -84,40 +84,40 @@ wxMultiCellItemHandle :: wxMultiCellItem
 }
 //---------------------------------------------------------------------------
-int wxMultiCellItemHandle::GetColumn()
+int wxMultiCellItemHandle::GetColumn() const
 {
     return m_column;
 }
 //---------------------------------------------------------------------------
-int wxMultiCellItemHandle::GetRow()
+int wxMultiCellItemHandle::GetRow() const
 {
     return m_row;
 }
 //---------------------------------------------------------------------------
-int wxMultiCellItemHandle::GetWidth()
+int wxMultiCellItemHandle::GetWidth() const
 {
     return m_width;
 }
 //---------------------------------------------------------------------------
-int wxMultiCellItemHandle::GetHeight()
+int wxMultiCellItemHandle::GetHeight() const
 {
     return m_height;
 }
 //---------------------------------------------------------------------------
-wxResizable    wxMultiCellItemHandle :: GetStyle()
+wxResizable    wxMultiCellItemHandle :: GetStyle() const
 {
     return m_style;
 };
 //---------------------------------------------------------------------------
-wxSize wxMultiCellItemHandle :: GetLocalSize()
+wxSize wxMultiCellItemHandle :: GetLocalSize() const
 {
     return m_fixedSize;
 };
 //---------------------------------------------------------------------------
-int wxMultiCellItemHandle :: GetAlignment()
+int wxMultiCellItemHandle :: GetAlignment() const
 {
     return m_alignment;
 };
 //---------------------------------------------------------------------------
-wxSize wxMultiCellItemHandle :: GetWeight()
+wxSize wxMultiCellItemHandle :: GetWeight() const
 {
     return m_weight;
@@ -511,5 +511,5 @@ void wxMultiCellSizer :: GetMinimums()
  */
 
-int wxMultiCellSizer :: Sum(int *array, int x)
+int wxMultiCellSizer :: Sum(int *array, int x) const
 {
     int sum = 0;

 	  	 
